README.md

```diff
@@ -63,7 +63,23 @@ Build the Surefire project using **Maven 3.2.5+** and **JDK 1.8+**.
   ```
   mvn install site site:stage -P reporting,run-its
   ```
-  
+
+* To set up the project in **Eclipse IDE** , please follow these steps:
+
+  * Maven-build (without tests) the `surefire-shared-utils`  module with profile `ide-development` and install it into the local maven repository
+    ```
+    cd surefire-shared-utils
+    mvn clean install -Pide-development -DskipTests
```

Comment:
The module does not have tests. More simple command line is `mvn install -P ide-development -f surefire-shared-utils/pom.xml`.